pkg/operator/controller/gatewayapi/status.go

```diff
@@ -0,0 +1,80 @@
+package gatewayapi
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"k8s.io/apimachinery/pkg/api/equality"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+
+	configv1 "github.com/openshift/api/config/v1"
+
+	operatorcontroller "github.com/openshift/cluster-ingress-operator/pkg/operator/controller"
+	"github.com/openshift/cluster-ingress-operator/pkg/operator/controller/status"
+)
+
+// setUnmanagedGatewayAPICRDNamesStatus sets the status of the ingress cluster operator
```

Comment:
nit:
```suggestion
// setUnmanagedGatewayAPICRDNamesStatus sets the status of the cluster ingress operator
```

Reply:
The resource is "clusteroperators", the kind is "ClusterOperator", and the specific object is named "ingress".  So "ingress clusteroperator" or "ingress ClusterOperator" would be fine.  (Quoting the name might make it more obvious: '"ingress" clusteroperator' or '"ingress" ClusterOperator'.)

Reply:
Let me quote the name of the cluster operator to reduce confusion.

Reply:
Done.